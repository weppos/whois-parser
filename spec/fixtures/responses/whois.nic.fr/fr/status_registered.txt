%%
%% This is the AFNIC Whois server.
%%
%% complete date format : YYYY-MM-DDThh:mm:ssZ
%% short date format    : DD/MM
%% version              : FRNIC-2.5
%%
%% Rights restricted by copyright.
%% See https://www.afnic.fr/en/products-and-services/services/whois/whois-special-notice/
%%
%% Use '-h' option to obtain more information about this service.
%%
%% [209.153.220.50 REQUEST] >> 1c2.fr
%%
%% RL Net [##########] - RL IP [#########.]
%%

domain:      1c2.fr
status:      ACTIVE
hold:        NO
holder-c:    U4339-FRNIC
admin-c:     MO2494-FRNIC
tech-c:      MO2556-FRNIC
zone-c:      NFC1-FRNIC
nsl-id:      NSL1231-FRNIC
registrar:   EPAG Domainservices GmbH
Expiry Date: 2019-05-07T07:43:45Z
created:     2004-05-20T11:28:40Z
last-update: 2018-04-08T03:52:04Z
source:      FRNIC

ns-list:     NSL1231-FRNIC
nserver:     ns.magic.fr
nserver:     ns2.magic.fr
source:      FRNIC

registrar:   EPAG Domainservices GmbH
type:        Isp Option 1
address:     Niebuhrstra√üe 16b
address:     DE-53113 BONN
country:     DE
phone:       +49 228 3296840
fax-no:      +49 228 3296849
e-mail:      support@epag.de
website:     https://www.epag.de/en/services/domains/domain-pricing/
anonymous:   NO
registered:  2006-01-11T12:00:00Z
source:      FRNIC

nic-hdl:     U4339-FRNIC
type:        ORGANIZATION
contact:     1C2
address:     20-22, rue Louis Armand
address:     75015 Paris
country:     FR
phone:       +33 1 30 62 40 06
e-mail:      jmr@1c2.com
registrar:   EPAG Domainservices GmbH
changed:     2016-09-22T07:52:47Z nic@nic.fr
anonymous:   NO
obsoleted:   NO
eligstatus:  ok
eligsource:  REGISTRY
eligdate:    2016-09-22T07:52:47Z
reachmedia:  email
reachstatus: ok
reachsource: REGISTRY
reachdate:   2016-09-22T07:52:47Z
source:      FRNIC

nic-hdl:     MO2494-FRNIC
type:        PERSON
contact:     Magic Online
address:     Magic Online
address:     130, avenue President Wilson
address:     93100 Montreuil
address:     Paris
country:     FR
phone:       +33 1 41 58 22 50
fax-no:      +33 1 56 72 93 30
e-mail:      hostmaster@magic.fr
registrar:   EPAG Domainservices GmbH
changed:     2014-03-31T16:13:52Z nic@nic.fr
anonymous:   NO
obsoleted:   NO
eligstatus:  not identified
reachstatus: not identified
source:      FRNIC

nic-hdl:     MO2556-FRNIC
type:        PERSON
contact:     Magic Online
address:     Magic Online
address:     130-134, avenue du President Wilson
address:     93100 Montreuil
address:     France
country:     FR
phone:       +33 1 41 58 22 50
fax-no:      +33 1 56 72 93 30
e-mail:      production@magic.fr
registrar:   EPAG Domainservices GmbH
changed:     2014-05-20T15:07:08Z nic@nic.fr
anonymous:   NO
obsoleted:   NO
eligstatus:  not identified
reachstatus: not identified
source:      FRNIC

