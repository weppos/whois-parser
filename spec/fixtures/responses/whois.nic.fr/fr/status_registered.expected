#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2000-07-27}

#updated_on
  %s %CLASS{time}
  %s %TIME{2016-12-30}

#expires_on
  %s %CLASS{time}
  %s %TIME{2017-12-30}


#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "GIH6-FRNIC"
  %s[0].name          == nil
  %s[0].organization  == "Google Ireland Holdings"
  %s[0].address       == "70 Sir John Rogersons Quay\n2 Dublin"
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == "IE"
  %s[0].phone         == "+353 14361000"
  %s[0].fax           == nil
  %s[0].email         == "dns-admin@google.com"
  %s[0].updated_on    %TIME{2015-03-20 00:00:00 UTC}

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "GIH5-FRNIC"
  %s[0].name          == nil
  %s[0].organization  == "Google Ireland Holdings"
  %s[0].address       == "70 Sir John Rogersons Quay\n2 Dublin"
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == "IE"
  %s[0].phone         == "+353 14361000"
  %s[0].fax           == nil
  %s[0].email         == "dns-admin@google.com"
  %s[0].updated_on    %TIME{2011-12-06 00:00:00 UTC}

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "CP4370-FRNIC"
  %s[0].name          == "Ccops Provisioning"
  %s[0].organization  == "MarkMonitor"
  %s[0].address       == "10400 Overland Rd.\nPMB 155\n83709 Boise"
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1 2083895740"
  %s[0].fax           == "+1 2083895771"
  %s[0].email         == "ccops@markmonitor.com"
  %s[0].updated_on    %TIME{2011-06-14 00:00:00 UTC}


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.google.com"
